lte/cloud/go/services/subscriberdb/digest.go

```diff
@@ -101,33 +101,33 @@ func GetPerSubscriberDigests(network string) ([]*lte_protos.SubscriberDigestWith
 // ordered by their subscriber IDs (unique within a network). It returns
 // 1. A set of subscribers that have been added/modified, with the new digests.
 // 2. An ordered list of subscribers that have been removed.
-func GetPerSubscriberDigestsDiff(all []*lte_protos.SubscriberDigestWithID, tracked []*lte_protos.SubscriberDigestWithID) (map[string]string, []string) {
-	n, m, i, j := len(all), len(tracked), 0, 0
+func GetPerSubscriberDigestsDiff(next []*lte_protos.SubscriberDigestWithID, prev []*lte_protos.SubscriberDigestWithID) (map[string]string, []string) {
```

Comment:
In the args and throughout the fn -- be consistent about "prev" first, then "next". It reads better that way